about.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>FABL:TEMPLATE</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="/css/styles.css" />
+    <link href="https://fonts.googleapis.com/css?family=Asar|Sansita" rel="styles.css">
+    <script src="main.js"></script>
+</head>
+    <body>
+        <header>
+            <img id="logo" src="images/logos/logo-FABL-horz_07.png">
+        <nav>
+        <ul>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">RubyRoute</a></li>
+            <li><a href="#">Products</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+        </nav>
+        </header>
+        <div>
+            <section id="">
+                <div id="">
+                <article>
+                        <h1>About FABL</h1>
+                        <h3>Who are we?</h3>
```

Comment:
Always follow an H1 with an H2 - we can always adjust the font size in the css later